specs/phase0/beacon-chain.md

```diff
@@ -219,6 +220,7 @@ The following values are (non-configurable) constants used throughout the specif
 | - | - | :-: | :-: |
 | `MIN_GENESIS_DELAY` | `86400` | seconds | 1 day |
 | `SECONDS_PER_SLOT` | `12` | seconds | 12 seconds |
+| `SECONDS_PER_ETH1_BLOCK` | `14` | seconds | |
```

Comment:
```suggestion
| `SECONDS_PER_ETH1_BLOCK` | `14` | seconds | 14 seconds |
```
It's not pretty, but at least consistent with the line above like this.

Reply:
lol that is a funny column